tests/mmptest/src/NetStandardTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Xamarin.MMP.Tests
+{
+	[TestFixture]
+	public partial class NetStandardTests
+	{
+		// [Test] https://bugzilla.xamarin.com/show_bug.cgi?id=53164
+		public void ModernUsingNetStandardLib_SmokeTest ()
+		{
+			MMPTests.RunMMPTest(tmpDir => {
+				NetStandardTestCore (tmpDir, false);
+			});
+		}
+
+		// [Test] https://bugzilla.xamarin.com/show_bug.cgi?id=53164
+		public void FullUsingNetStandardLib_SmokeTest ()
+		{
+			MMPTests.RunMMPTest(tmpDir => {
+				NetStandardTestCore (tmpDir, true);
+			});
+		}
+
+		static void NetStandardTestCore (string tmpDir, bool full)
+		{
+			TI.UnifiedTestConfig config = new TI.UnifiedTestConfig (tmpDir);
+
+			string netStandardProject = TI.GenerateNetStandardProject (config);
+			TI.RunAndAssert("/usr/local/share/dotnet/dotnet", $"restore {netStandardProject}", "Restore");
```

Comment:
Maybe check if `/usr/local/share/dotnet/dotnet` exists, and if not ignore the test? I don't have it on my system.

Reply:
Great point.